draft/spec/index.html

```diff
@@ -338,7 +338,9 @@ <h2>Object Structure</h2>
     ├── inventory.json.sha512
     └── v1
         ├── inventory.json
-        └── inventory.json.sha512
+        ├── inventory.json.sha512
+        └── content
+            └── .keep
```

Comment:
I do not think we should show a `content` dir with a `.keep`.  Technically, the `content` and inventory files in the version should not be present anyway because the current description says "The **minimum required** is." -- neither are required for the minimum. However, I think it would be better to instead say something like:

> The OCFL Object structure organizes content files and administrative information in order to support content storage and object validation. The structure for an object with one version is shown in the following figure:
>
> ```
> [object_root]
>    ├── 0=ocfl_object_1.0
>    ├── inventory.json
>    ├── inventory.json.sha512
>    └── v1
>        ├── inventory.json
>        ├── inventory.json.sha512
>        └── content
>            └── ... content files ...

Reply:
Somehow this got marked outdated in a race condition --- I suggest also that we must reword the intro text because this isn't the minimal form

Reply:
WIll do.